README.md

```diff
@@ -19,12 +19,9 @@ This filter initializes an HTTP header(X-B3-TraceID) for tracing, if not already
 * Optionally, this filter can also be used to generate audit events which includes the request and response payload.
 
 ### Example Log4j pattern to use tracing/vcap info in logs
-Add the log pattern to the log4j.properties file. Then reference the log pattern as the layout's conversion pattern using the EnhancedPatternLayout layout for desired appenders.
+Reference the PredixLayout for desired appenders.
```

Comment:
* Change the section heading to something like:   "Provide a log4j JSON layout which also logs VCAP information."
* add a example log message